docs/intro.asciidoc

```diff
@@ -36,87 +25,7 @@ They make it easy to format your logs into ECS-compatible JSON. For example:
 
-[float]
-=== Why ECS logging?
-
-*No parsing of the log file required*::
-+
---
-ECS-compatible JSON doesn't require the use of Logstash or grok parsing via an ingest node pipeline.
---
-
-*No external dependencies*::
-+
---
-By not using any external dependencies such as JSON serializers,
-the library is incredibly lightweight.
---
-
-*Highly efficient*::
-+
---
-The log4j2 `EcsLayout` does not allocate any memory (unless the log event contains an `Exception`) which reduces GC pressure.
-This is achieved by manually serializing JSON so that no intermediate JSON or map representation of a log event is needed.
---
-
-*Decently human-readable JSON structure*::
-+
---
-The first three fields are always `@timestamp`, `log.level` and `message`.
-It's also possible to format stack traces so that each element is rendered in a new line.
---
-
-*Enjoy the benefits of a common schema*::
-+
---
-Use the Kibana {observability-guide}/monitor-logs.html[Logs app] without additional configuration.
-
-Using a common schema across different services and teams makes it possible create reusable dashboards and avoids {ref}/mapping.html#mapping-limit-settings[mapping explosions].
---
-
-*APM Log correlation*::
-+
---
-If you are using the {apm-java-ref}/index.html[Elastic APM Java agent],
-you can leverage the {apm-java-ref}/config-logging.html#config-enable-log-correlation[log correlation feature] without any additional configuration.
-This lets you jump from the {kibana-ref}/spans.html[Span timeline in the APM UI] to the {observability-guide}/monitor-logs.html[Logs app],
-showing only the logs which belong to the corresponding request.
-Vice versa, you can also jump from a log line in the Logs UI to the Span Timeline of the APM UI.
---
-
-[float]
-==== Additional advantages when using in combination with Filebeat
-
-We recommend using this library to log into a JSON log file and using Filebeat to send the logs to Elasticsearch. Here are a few benefits to this approach.
-
-*Resilient in case of outages*::
-+
---
-{filebeat-ref}/how-filebeat-works.html#at-least-once-delivery[Guaranteed at-least-once delivery]
-without buffering within the application, thus no risk of `OutOfMemoryError` s or lost events.
-There's also the option to use either the JSON logs or plain-text logs as a fallback.
---
-
-*Loose coupling*::
-+
---
-The application does not need to know the details of the logging backend (URI, credentials, etc.).
-You can also leverage alternative {filebeat-ref}/configuring-output.html[Filebeat outputs],
-like Logstash, Kafka or Redis.
---
-
-*Index Lifecycle management*::
-+
---
-Leverage Filebeat's default {filebeat-ref}/ilm.html[index lifecycle management settings].
-This is much more efficient than using daily indices.
---
-
-*Efficient Elasticsearch mappings*::
-+
---
-Leverage Filebeat's default ECS-compatible {filebeat-ref}/configuration-template.html[index template]
---
+Ready to jump in? See the <<setup,get started>> guide.
 
 [float]
 === Mapping
```

Comment:
Can this section move to the `Structured logging with log4j2` page?

Reply:
I've made it generic and added it to the overview docs. See https://github.com/elastic/ecs-logging/pull/40/commits/c1d11d1c126f27aca375307c0f6d0b7d8b2b2bd3.